package.json

```diff
@@ -29,34 +29,47 @@
     "Cloud Asset API"
   ],
   "scripts": {
-    "docs": "jsdoc -c .jsdoc.js",
-    "lint": "eslint '**/*.js'",
     "samples-test": "c8 npm run samples-test-suite",
     "samples-test-suite": "cd samples/ && npm link ../ && npm test && cd ../",
-    "system-test": "mocha system-test/*.js smoke-test/*.js --timeout 600000",
-    "test": "c8 mocha",
-    "fix": "eslint --fix '**/*.js'",
+    "clean": "gts clean",
+    "compile": "tsc -p . && cp -r protos build/",
+    "compile-protos": "compileProtos src",
+    "docs": "jsdoc -c .jsdoc.js",
     "docs-test": "linkinator docs",
-    "predocs-test": "npm run docs"
+    "fix": "gts fix",
+    "lint": "gts check",
+    "predocs-test": "npm run docs",
+    "prepare": "npm run compile-protos && npm run compile",
```

Comment:
We don't usually do `compile-protos` in `prepare`, it makes the install step more error-prone. We normally just do `npm run compile` here and run `npx compileProtos src` in `synth.py`, that way the protos are already checked in and there is no need to recompile them every time.